### DISCUSSION TITLE
关于IC2中无法与<工业工作台>进行交互bug

### DISCUSSION THREAD

**Discussion by WaterDouble** [IndustrialCraftMC/IndustrialCraft2#37] [【漏洞反馈】Bugs Report]
使用模组版本：1.industrialcraft-2-2.9.84+ex119-1.19.2-forge
2.[1.19.2]Jei物品管理器(forge11.3.0.262)

游戏版本：1.19.2 forge-43.2.0 Op-I1-pre4

问题描述：游戏中无法与<工业工作台>进行交互。

**Comment by AbbottMc** [comment]
已修复
